els.go

```diff
@@ -172,18 +249,50 @@ func indexELSData(xmlMap []map[string]interface{}, host string, index string, da
 		}
 
 		jsonData = jsonData + "}"
-		bulkService.Add(elastic.NewBulkIndexRequest().Index(index).Type(dataType).Id(id).Doc(jsonData))
+		jsonData = strings.Replace(jsonData,"\t"," ",-1)
+
+        //fmt.Printf("%s\n", jsonData)
+		//f.Write([]byte(jsonData+"\n"))
+		bulkService.Add(elastic.NewBulkIndexRequest().Id(id).Doc(jsonData))
+		fmt.Printf("%+v",bulkService.NumberOfActions())
 		if bulkService.NumberOfActions() >= bulkSize {
-			_, err = bulkService.Do(ctx)
+			bulkResponse, err := bulkService.Do(ctx)
+			indexed := bulkResponse.Indexed()
+			fmt.Printf("%+v,",indexed[0])
 			if err != nil {
 				log.Error("Failed in doing bulk request ", err)
 			}
 		}
 	}
 	// ingest the rest of the data
-	_, err = bulkService.Do(ctx)
-	if err != nil {
-		log.Error("Failed in doing bulk request ", err)
+	if bulkService.NumberOfActions() > 0 {
+		bulkResponse, err := bulkService.Do(ctx)
+		fmt.Printf("Last%+v,",bulkResponse.Failed())
```

Comment:
although the previous line is not catching anything in "err" but this line shows "Errors True" with 400 in the string, so i am assuming it is error 400 which i am not sure how to resolve?